### PR TITLE
Redone the new project form

### PR DESCRIPTION
**Pull request by Tiago-Palhoca**
-added the categories array
-changed the label for total_funding
-added a validates to the project model

### CODE REVIEW

File: app/models/project.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@ class Project < ApplicationRecord
 
   validates :name, :category, :total_funding, :total_shares, :owner_id, presence: true
   validates :name, uniqueness: true
+  validates :category, inclusion: { in: ["Eliminate Poverty", "Erase Hunger", "Establish Good Health and Well-Being", "Provide Quality Education", "Enforce Gender Equality", "Improve Clean Water and Sanitation", "Grow Affordable and Clean Energy", "Create Decent Work and Economic Growth", "Increase Industry, Innovation, and Infrastructure", "Reduce Inequality", "Mobilize Sustainable Cities and Communities", "Influence Responsible Consumption and Production", "Organize Climate Action", "Develop Life Below Water", "Advance Life On Land", "Guarantee Peace, Justice, and Strong Institutions", "Build Partnerships for the Goals"] }
```
Comment (raffoz) [inline app/models/project.rb:8]: @kleyv must be in seeds these categories

Comment (kleyv) [reply app/models/project.rb:8]: On it.

Comment (raffoz) [reply app/models/project.rb:8]: @kleyv ok you merge it after seed

**Comment by raffoz** [comment]
@kleyv when finish seed please merge this PR

**Pull request merged**